### PR TITLE
[v7r2] removed the watchdog for mac, simplified linux

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*wms
CHANGE: removed the watchdog for mac, simplified linux

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Please review.

File: WorkloadManagementSystem/JobWrapper/Watchdog.py [hunk 1]
```diff
@@ -925,10 +925,20 @@ def __setJobParamList(self, value):
...
+    """
+    result = {}
+    result["HostName"] = socket.gethostname()
+    result["CPU(MHz)"] = psutil.cpu_freq()[0]
+    result["Memory(kB)"] = int(psutil.virtual_memory()[1] / 1024)
+    result["LocalAccount"] = getpass.getuser()
+
+    with open("/proc/cpuinfo", "r") as cpuinfo:
+      info = cpuinfo.readlines()
+    result["ModelName"] = [x.strip().split(":")[1] for x in info if "model name" in x][0].strip()
```
Comment (andresailer) [inline WorkloadManagementSystem/JobWrapper/Watchdog.py:938]: In the previous version, all whitespace was replaced, not just at the edges, but if this is only used in the list of JobParameters keeping the internal whitespace is easier to read.
```python
result["ModelName"] = info[4].split(':')[1].replace(' ', '').replace('\n', '')
```

File: WorkloadManagementSystem/JobWrapper/Watchdog.py [hunk 2]
```diff
@@ -940,9 +950,17 @@ def getMemoryUsed(self):
 
   #############################################################################
   def getDiskSpace(self):
-    """ Attempts to get the available disk space, should be overridden in a subclass"""
-    methodName = 'getDiskSpace'
-    self.log.warn('Watchdog: ' + methodName + ' method should be implemented in a subclass')
-    return S_ERROR('Watchdog: ' + methodName + ' method should be implemented in a subclass')
+    """Obtains the disk space used.
```
Comment (andresailer) [inline WorkloadManagementSystem/JobWrapper/Watchdog.py:953]:
```suggestion
    """Obtains the available disk space.
```

File: WorkloadManagementSystem/JobWrapper/Watchdog.py [hunk 3]
```diff
@@ -940,9 +950,17 @@ def getMemoryUsed(self):
...
+    """
+    result = S_OK()
+    diskSpace = getDiskSpace()
+
+    if diskSpace == -1:
+      result = S_ERROR('Could not obtain disk usage')
+      self.log.warn(' Could not obtain disk usage')
+      result['Value'] = -1
```
Comment (andresailer) [inline WorkloadManagementSystem/JobWrapper/Watchdog.py:961]: This didn't change, but this value is just overwritten in line 963 (with `float(-1)` if this branch is taken, so nothing wrong at the moment, but it smells), the line could just be removed, or a `return result` after this line to be independent of what happens after the if.
```suggestion
      result['Value'] = float(-1)
      return result
```

Comment (fstagni) [reply WorkloadManagementSystem/JobWrapper/Watchdog.py]: Implemented this and the other suggestions.

**Comment by atsareg** [comment]
Conflicts to be resolved

**Pull request merged**